spec/integration/recipes/local.rb

```diff
@@ -8,3 +8,14 @@
   version '2.0.0'
   options ['--no-document']
 end
+
+######
+
+# Docker backend raises an error with `user` option, so it tests only on `itamae local`.
+# After fix this error, please move this code and the spec to `default.rb`.
```

Comment:
I tried to add the following code to `recipes/default.rb`, but it does not work.
Because docker backend raises an error with this code.
Docker backend's `send_file` sends a file with the root user, and currently, it is not configurable, unfortunately.
I guess we can move the code to `recipes/default.rb` by fixing docker backend, but this pull request does not focus this problem. I guess this problem already exists before this pull request because I do not touch docker backend in this pull request.